src/languages/html.coffee

```diff
@@ -0,0 +1,681 @@
+helper = require '../helper.coffee'
+parser = require '../parser.coffee'
+
+parse5 = require '../../vendor/parse5/index.js'
```

Comment:
@sakagg -- you said that the parser was slightly modified; what are the modifications?